tests/Kernel/ConsultationRenderTest.php

```diff
@@ -0,0 +1,187 @@
+            ],
+            [
+              'label' => 'Deadline',
+              'body' => '21 February 2020, 01:00 (AEDT)',
+            ],
+          ],
+        ]),
+      ],
+    ];
+    $assert->assertPattern($expected_values, $html);
+
+    $crawler = new Crawler($html);
+    $actual = $crawler->filter('span.call-status.ecl-label.ecl-u-text-uppercase.ecl-label--high.ecl-u-type-color-black');
+    $this->assertCount(1, $actual);
+
+    // Check status Closed label and background.
+    $deadline_date->modify('- 4 days');
+    $node->set('oe_consultation_deadline', [
+      $deadline_date->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT),
+    ])->save();
+    $build = $this->nodeViewBuilder->view($node, 'teaser');
+    $html = $this->renderRoot($build);
+    $expected_values['meta'] = 'Status: Closed';
+    $expected_values['additional_information'] = [
+      new PatternAssertState(new FieldListAssert(), [
+        'items' => [
+          [
+            'label' => 'Opening date',
+            'body' => '14 February 2020',
+          ], [
+            'label' => 'Deadline',
+            'body' => '17 February 2020, 12:00 (AEDT)',
+          ],
+        ],
+      ]),
+    ];
+    $assert->assertPattern($expected_values, $html);
+
+    $crawler = new Crawler($html);
+    $actual = $crawler->filter('span.call-status.ecl-label.ecl-u-text-uppercase.ecl-label--low.ecl-u-type-color-black');
+    $this->assertCount(1, $actual);
+
+    // Check status Upcoming label and background.
+    $opening_date->modify('+ 10 days');
+    $deadline_date->modify('+ 4 days');
+    $node->set('oe_consultation_deadline', [
+      $deadline_date->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT),
+    ]);
+    $node->set('oe_consultation_opening_date', $opening_date->format(DateTimeItemInterface::DATE_STORAGE_FORMAT))->save();
+    $build = $this->nodeViewBuilder->view($node, 'teaser');
+    $html = $this->renderRoot($build);
+    $expected_values['meta'] = 'Status: Upcoming';
+    $expected_values['additional_information'] = [
+      new PatternAssertState(new FieldListAssert(), [
+        'items' => [
+          [
+            'label' => 'Opening date',
+            'body' => '24 February 2020',
+          ], [
+            'label' => 'Deadline',
+            'body' => '21 February 2020, 12:00 (AEDT)',
+          ],
+        ],
+      ]),
+    ];
+    $assert->assertPattern($expected_values, $html);
+
+    $crawler = new Crawler($html);
+    $actual = $crawler->filter('span.call-status.ecl-label.ecl-u-text-uppercase.ecl-label--medium.ecl-u-type-color-black');
+    $this->assertCount(1, $actual);
+
+    // Check status N/A.
+    $node->set('oe_consultation_opening_date', NULL);
+    $node->set('oe_consultation_deadline', NULL);
+    $node->save();
+    $build = $this->nodeViewBuilder->view($node, 'teaser');
+    $html = $this->renderRoot($build);
+    $expected_values['meta'] = '';
+    $expected_values['additional_information'] = NULL;
```

Comment:
Not sure that this test is needed because oe_consultation_opening_date and oe_consultation_deadline are required.

I've tried this use case manually locally and div.ecl-content-item__additional_information isn't shown. But, indeed, in the test `<div class="ecl-content-item__additional_information ecl-u-mb-s"></div>` is printed but it shouldn't. I think the problem is not in ListItemAssert.